src/Aspire.Hosting.GitHub.Models/GitHubModelsResource.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.GitHub.Models;
+
+/// <summary>
+/// Represents a GitHub Models resource.
+/// </summary>
+public class GitHubModelsResource : Resource, IResourceWithConnectionString
+{
+    /// <summary>
+    /// The default endpoint for GitHub Models.
+    /// </summary>
+    public const string DefaultEndpoint = "https://models.github.ai/inference";
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="GitHubModelsResource"/> class.
+    /// </summary>
+    /// <param name="name">The name of the resource.</param>
+    /// <param name="model">The model name.</param>
+    public GitHubModelsResource(string name, string model) : base(name)
+    {
+        Model = model;
+        Endpoint = DefaultEndpoint;
+    }
+
+    /// <summary>
+    /// Gets or sets the endpoint URL for the GitHub Models service, e.g., "https://models.github.ai/inference".
+    /// </summary>
+    public string Endpoint { get; set; }
```

Comment:
This should be hardcoded.

Reply:
Agreed, there's no point in being able to configure this value.